src/frontend/src/components/getApartments.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+function getApartments(query: string, pageNum: number, selected: string[]) {
+   const [loading, setLoading] = useState(true);
+   const [error, setError] = useState(false);
+   const array: {
+      // avoids linting warnings
+      name: string;
+      address: string;
+      image: string;
+      review: string;
```

Comment:
The review(s) ans image(s) are handled by the right half, this list of apartments only need to store what is in the figma design.

Reply:
I know the review will be removed but we still need to image

Reply:
so you're saying I should store the images in a different object?